src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,23 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Transaction transaction;
+        try (Session session = factory.openSession()) {
+            transaction = session.beginTransaction();
+            session.persist(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            throw new DataProcessingException("Can't save actor of " + actor, e);
+        }
+        return actor;
```

Comment:
you should use try catch finally here and in finally make a transaction's rollback, the same for add methods in other DAOs